source/adios2/engine/sst/SstParamParser.cpp

```diff
@@ -10,6 +10,8 @@ using namespace adios2::core;
 
 void SstParamParser::ParseParams(IO &io, struct _SstParams &Params)
 {
+    memset(&Params, 0, sizeof(Params));
+
```

Comment:
`memset` -> `std::memset`.  We like to use the C++ interface to the C library rather than the C library directly.

Reply:
Will update in the next push.